[Browser Isolation] Added network dependencies (#4303)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/browser-isolation/network-dependencies.md

---
pcx-content-type: reference
title: Browser Isolation with firewall
weight: 9
---

# Browser Isolation with firewall

> If your organization uses a firewall or other policies to restrict Internet traffic, you may need to make a few changes to allow Browser Isolation to connect.

Reviewer: "may" is a bit vague. How do users know which steps they need to take? Should we recommend that they configure all of the permissions listed below?

> ## Remoting client

Reviewer: is Remoting client different from the WARP client?

Author: Yes, it's the web based client we serve to the local web browser that connects to the remote browser.

> Isolated pages are served the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.

Reviewer: ```suggestion
Isolated pages are served by the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.
```

### Remoting Client (Services)

> Provides static assets and API endpoints required for Browser Isolation to function.

Reviewer: ```suggestion
The remoting client provides static assets and API endpoints required for Browser Isolation to function.
```

> - Allow HTTPS traffic to `*.browser.run` on port 443

Reviewer: ```suggestion
- HTTPS traffic to `*.browser.run` on port 443
```

> - HTTPS traffic to `*.browser.run` on port 443

Reviewer: ```suggestion
- HTTPS traffic to `*.browser.run` on port `443`
```

#### Clientless Web Isolation

> Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access:

Reviewer: ```suggestion
Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access. For users to connect to Access, you must allow:
```

> - Allow HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443

Reviewer: ```suggestion
- HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
```

> - HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443

Reviewer: ```suggestion
- HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port `443`
```

### WebRTC channel

> Browse Isolation uses WebRTC for low-latency communication between the local browser and the remote browser. 

Reviewer: ```suggestion
Browser Isolation uses WebRTC for low-latency communication between the local browser and the remote browser. 
```

> These are the IP addresses that the Remoting Client will connect to. All WebRTC traffic from your device to the remote browser will go through these IP addresses.

Reviewer: ```suggestion
In order to pass WebRTC traffic, the remoting client must be able to connect to the following IP addresses:
```
